config/webpack/utils/cssInJsLoader.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable */
```

Comment:
If we are opting out of linting because the source is from elsewhere, might be handy to have a link back to the original source?